### PR TITLE
Enable templates in scheduled maintenance dashboard

### PR DESCRIPTION
**Pull request by shuichiro-makigaki**
plus clean up

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Thanks for this! I'm going to review over the weekend!

**Comment by shuichiro-makigaki** [comment]
@jbrooksuk ping

**Comment by shuichiro-makigaki** [comment]
@jbrooksuk ping?

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /node_modules
 /public/hot
 /public/storage
+/public/dist
```
Comment (jbrooksuk) [inline .gitignore]: Please don't commit these files.

Comment (shuichiro-makigaki) [reply .gitignore]: Revert previous clean-up commit

**Comment by shuichiro-makigaki** [comment]
@jbrooksuk ping

**Comment by jbrooksuk** [comment]
Thanks @shuichiro-makigaki!

**Pull request merged**